src/Modules/SharedModule/components/Navbar/Navbar.tsx

```diff
@@ -1,3 +1,565 @@
-export default function Navbar() {
-  return <div>Navbar</div>;
+import MenuIcon from "@mui/icons-material/Menu";
+import {
+  Avatar,
+  Box,
+  Button,
+  Divider,
+  Drawer,
+  IconButton,
+  List,
+  ListItem,
+  ListItemButton,
+  ListItemText,
+  Toolbar,
+  Tooltip,
+  Typography,
+  styled,
+  useTheme,
+} from "@mui/material";
+// import MenuItem from "@mui/material/MenuItem";
+import { useAuth } from "../../../../Context/AuthContext/AuthContext";
+
+import { FormatAlignCenter, KeyboardArrowDown } from "@mui/icons-material";
+import MuiAppBar from "@mui/material/AppBar";
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { AppBarProps } from "../../../../Interfaces/interFaces";
+import logoDark from "../../../../assets/images/logo-dark.svg";
+import logoLight from "../../../../assets/images/logo-light.svg";
+
+export default function Navbar({
+  setTheme,
+  setOpen,
+  open,
+  window,
+}: AppBarProps) {
+  const { loginData, logOut } = useAuth();
+  const navigate = useNavigate();
+  const [profileMenu, setProfileMenu] = useState(false);
+  const [isDark, setIsDark] = useState(() => {
+    const value = localStorage.getItem("theme");
+    if (value === "dark" || value === null) return true;
+    return false;
+  });
+  const navBtns = (
+    <Box
+      display={"flex"}
+      justifyItems={"center"}
+      alignItems={"center"}
+      gap={2}
+      flexDirection={{ xs: "column", md: "row" }}
+    >
+      <Button
```

Comment:
This should be a Link Component, we can give the Button a component Prop and give it the Link Component and then instead of having an onClick, we can use the `to` prop and give it the path.